2020-01-Winter/3_stacks_queues/recent_counter/recent_counter_python.py

```diff
@@ -0,0 +1,39 @@
+class RecentCounter:
+    #This queue is a Python list
+    #In this problem, we treat it like a FIFO Queue
+    queue = []
```

Comment:
This is fine for a single example, but the variable `queue` is being treated as static to the class instead of specific to the instance. If I try to add new examples / class instances, the contents of the variable are retained, skewing the output for subsequent tests.
```suggestion
    def __init__(self):
        self.queue = []
```

Reply:
I forgot my basics 😢. I'll change it